requirements.txt

```diff
@@ -22,4 +22,8 @@ wheel==0.36.2  # dep install
 
 # pre-commit & formatting deps
 pre-commit==2.17.0
-black==22.1.0
\ No newline at end of file
+black==22.1.0
+
+# Fixing for JKinja Requirements
+jinja2<3.1.0
+werkzeug==2.0.3
```

Comment:
Looks like we already have werkzeug pinned on L15 so this is what's causing the test failures